src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/EmbeddedLanguages/VirtualChars/CSharpVirtualCharService.cs

```diff
@@ -41,7 +41,7 @@ protected override bool IsMultiLineRawStringToken(SyntaxToken token)
         return false;
     }
 
-    protected override VirtualCharSequence TryConvertToVirtualCharsWorker(SyntaxToken token)
+    protected override VirtualCharGreenSequence TryConvertToVirtualCharsWorker(SyntaxToken token)
```

Comment:
the languages are now just responsible for creating the green sequence.  which can be done just with the Kind and ValueText of a token.  This still references SyntaxToken since this is currently at teh Workspace level.  But if/when this moves to the compiler, this can/should reference just the green node.